code/datums/viewport.dm

```diff
@@ -0,0 +1,100 @@
+#define ceil(x) (-round(-(x)))
```

Comment:
We definitely already have this in a define (probably `CEIL`), and if we don't, then add it to math helpers.

Reply:
CEILING